### PR TITLE
Adding CI files 2

### PR DESCRIPTION
**Pull request by Shahar-Fleischman-Forter**
asana: https://app.asana.com/0/23095211578528/1208259038709552

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+]
+
+node('general') {
+    def base = load '/var/jenkins_home/workspace/Infra/build-scripts/build/Jenkinsfile'
+    base.execute(
+        distImages: ['forter-gist'],
+	customStages: stages
+        credentials: base.get_decrypt_creds(),
+        linterReportLocation: 'test-reports/ci-lint-report',
+        masterBranch: 'main',
```
Comment (Mohamad-Khaleqi-Forter) [inline Jenkinsfile:17]: there is no `main` branch. still old convention `master`

Comment (Jeremy-Marer-Forter) [reply Jenkinsfile]: changed

Comment (Shahar-Fleischman-Forter) [reply Jenkinsfile]: Fixing

**Review by Mohamad-Khaleqi-Forter — COMMENTED** [review]
LGTM

**Pull request merged**